test/integration/patients/list/patients-all.js

```diff
@@ -76,4 +76,57 @@ context('patient all list', function() {
       .get('.list-page__filters')
       .contains('Another Group');
   });
+
+  specify('name sorting', function() {
+    cy
+      .server()
+      .routeGroups(_.indentity, testGroups)
+      .routePatient()
+      .routePatientActions()
+      .routePatients(fx => {
+        fx.data = _.sample(fx.data, 3);
+
+        fx.data[0].id = '1';
```

Comment:
is it necessary to set the `id`s?

Reply:
maybe not all of them...but I was having some issues with the routes not being found on some of them. I can poke around and pull out what I don't need

Reply:
yeah I think only on items that test either a click or a data update, but anything that's display only shouldn't need it.